### ISSUE TITLE
[angular, jest] Please update packages

### ISSUE THREAD

**Issue by snebjorn** [nrwl/nx#2686]
## Expected Behavior

Packages should be up to date.

## Current Behavior

Fixed versions that need updating. Why is it fixed? a `~version` seems reasonable.
```
 @angular/animations                   9.0.6  →     9.0.7 
 @angular/common                       9.0.6  →     9.0.7
 @angular/compiler                     9.0.6  →     9.0.7
 @angular/core                         9.0.6  →     9.0.7
 @angular/forms                        9.0.6  →     9.0.7
 @angular/platform-browser             9.0.6  →     9.0.7
 @angular/platform-browser-dynamic     9.0.6  →     9.0.7
 @angular/router                       9.0.6  →     9.0.7
 @angular/language-service             9.0.6  →     9.0.7
 @types/jest                          24.9.1  →    25.1.4
 dotenv                                6.2.0  →     8.2.0
 jest                                 24.9.0  →    25.1.0
 jest-preset-angular                   8.0.0  →     8.1.2
 ts-jest                              24.0.0  →    25.2.1
```
`~`Minor versions that need updating.
```
 ts-node                              ~7.0.0  →    ~8.7.0
 @types/node                          ~8.9.4  →   ~13.9.2
```

These are up to date. But could use a version bump.
```
 @angular-devkit/build-angular      ~0.900.6  →  ~0.900.7
 @angular-devkit/build-ng-packagr   ~0.900.6  →  ~0.900.7
 @angular/compiler-cli                ~9.0.6  →    ~9.0.7
 codelyzer                            ~5.0.1  →    ~5.2.1
 zone.js                             ^0.10.2  →   ^0.10.3
 commitiquette                        ^1.0.2  →    ^1.0.3
 cypress                              ^4.1.0  →    ^4.2.0
 ng-packagr                           ^9.0.0  →    ^9.0.3
 tslint                               ~6.0.0  →    ~6.1.0
```

## Other

I wouldn't mind making a PR but I couldn't find the relevant places. So if someone could point me in the right direction.

**Comment by mehrad-rafigh** [comment]
Hi @snebjorn 
I updated my workspace to 9.1.2 and the ng dependencies are up to date.

jest-preset-angular needs an update to 8.1.2. I will create a PR today to address that :)
Be safe!

**Comment by snebjorn** [comment]
FYI angular just released 9.1 :)

https://blog.angular.io/version-9-1-of-angular-now-available-typescript-3-8-faster-builds-and-more-eb292f989428

**Comment by snebjorn** [comment]
@mehrad-rafigh I was trying to figure out how you updated the dependencies so I could contribute in the future.

But I cannot find any relevant package.json changes here https://github.com/nrwl/nx/compare/9.1.0...9.1.3

Perhaps you can enlighten me :)

**Comment by Sidewinder1138** [comment]
In my fresh nx 9.1.3 workspace, the codelyzer v5.0.1 is complaining that it wants @angular/compiler@>=2.3.1 <9.0.0 || >8.0.0-beta <9.0.0, so it looks like it doesn't work with Angular 9... Can we bump codelyzer up to a version that does?

**Comment by snebjorn** [comment]
codelyzer just released [5.2.3](https://github.com/mgechev/codelyzer/blob/master/CHANGELOG.md#523-2020-03-26) which have support for TSLint >6 (that includes 6.1.0) 🎉
This is great!

**Comment by brandonroberts** [comment]
Closing as resolved
